test/unit/browser/pages/vscode.test.ts

```diff
@@ -2,15 +2,31 @@
  * @jest-environment jsdom
  */
 import { JSDOM } from "jsdom"
+import { InternalNLSConfiguration } from "../../../../lib/vscode/src/vs/base/node/languagePacks"
```

Comment:
Nice! When I tried this a while back, somebody (TypeScript or ESLint) didn't like it, but glad to see you got it working 👏